src/Utility/Profiler.h

```diff
@@ -57,7 +57,8 @@ class Profiler
   void stop(const std::string& funcName);
 
   //! \brief Prints a profiling report for all tasks that have been measured.
-  void report(std::ostream& os) const;
+  template<class Stream>
+  void report(Stream& os) const;
```

Comment:
Prefer to have the template declaration on the same line, when everything fits within the '80 column standard"